src/components/devsupport/components/measure/measure.component.tsx

```diff
@@ -4,12 +4,14 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  */
 
-import React from 'react';
+import React, { useContext } from 'react';
 import {
   findNodeHandle,
   UIManager,
+  StatusBar,
 } from 'react-native';
 import { Frame } from './type';
+import { ApplicationContext } from '../../../theme/application/applicationProvider.component';
```

Comment:
Actually, it's not a good idea, to import appProvider into some utility component.
I'd rather add to `MeasureComponent` new prop to calculate with insets and then handle the calculations inside.